### PR TITLE
Fix _MergedWrapperMarker to not include markers without corresponding execution scripts

### PR DESCRIPTION
**Pull request by BruceForstall**
Don't include markers without execution scripts

### CODE REVIEW

**Comment by BruceForstall** [comment]
I verified that the only groups missing from a Pri-0 Linux-x64 test run now are:

HardwareIntrinsics_Arm_r
HardwareIntrinsics_Arm_ro

which is what we want, since they were the ones failing before.

**Comment by BruceForstall** [comment]
Failure in outerloop is https://github.com/dotnet/runtime/issues/85081 (which is supposed to be disabled).

File: src/tests/Common/helixpublishwitharcade.proj [hunk 1]
```diff
@@ -22,15 +22,15 @@
     ============================================
 
   <PropertyGroup>
-    <BUILD_SOURCESDIRECTORY>c:\gh\runtime</BUILD_SOURCESDIRECTORY>
-    <CorrelationPayloadDirectory>c:\bugs\spmicollect3\payload\correlation</CorrelationPayloadDirectory>
-    <WorkItemDirectory>c:\bugs\spmicollect3\payload\workitem</WorkItemDirectory>
+    <BUILD_SOURCESDIRECTORY>/home/brucefo/gh/runtime</BUILD_SOURCESDIRECTORY>
+    <CorrelationPayloadDirectory>/home/brucefo/test/correlation</CorrelationPayloadDirectory>
+    <WorkItemDirectory>/home/brucefo/test/workitem</WorkItemDirectory>
```
Comment (ivdiazsa) [inline src/tests/Common/helixpublishwitharcade.proj:27]: Is this going to be reverted, or is it necessary for the PR?

**Comment by ivdiazsa** [comment]
I've a question I left in a comment, but otherwise it LGTM! Thanks for fixing this Bruce

**Review by ivdiazsa — APPROVED** [review]

File: src/tests/Common/helixpublishwitharcade.proj (hunk 1 above)
Comment (BruceForstall) [reply src/tests/Common/helixpublishwitharcade.proj:27]: This is commented-out code, so it can stay as an example for future users.

**Comment by BruceForstall** [comment]
fwiw, on a linux-x64 dev box you might build using:
```
	./build.sh -subset clr+libs -arch x64 -c checked
	./src/tests/build.sh checked x64
```

But on the CI, the build process is more like this (with many things happening on separate machines):
```
	./build.sh -subset clr.nativeprereqs -arch x64  -c checked
	./src/coreclr/build-runtime.sh checked x64 --keepnativesymbols
	./build.sh -subset tools+libs+libs.tests -configuration Debug -arch x64 -framework net8.0  -testscope innerloop /p:ArchiveTests=true
	./build.sh -subset clr.corelib+clr.nativecorelib+clr.nativeaotlibs+clr.tools+clr.packages+clr.paltestlist -arch x64 -c checked
	./src/tests/build.sh -log:Managed allTargets skipnative skipgeneratelayout skiptestwrappers checked x64 /p:TargetOS=AnyOS
	mkdir ~/gh/runtime/artifacts/tests/coreclr/linux.x64.Checked
	cp -r ~/gh/runtime/artifacts/tests/coreclr/AnyOS.x64.Checked/* ~/gh/runtime/artifacts/tests/coreclr/linux.x64.Checked
	./src/tests/build.sh skipmanaged skipgeneratelayout checked x64
	./src/tests/build.sh copynativeonly -log:Native checked x64 /p:LibrariesConfiguration=Debug
	./src/tests/build.sh buildtestwrappersonly -log:Wrappers checked x64 /p:LibrariesConfiguration=Debug
	./src/tests/build.sh generatelayoutonly -log:Layout checked x64 /p:LibrariesConfiguration=Debug
```

**Review by trylek — COMMENTED** [review]
Thanks for landing the real fix Bruce!

**Pull request merged**